lib/lambda_ethereum_consensus/p2p/gossip/attestation.ex

```diff
@@ -9,16 +9,19 @@ defmodule LambdaEthereumConsensus.P2P.Gossip.Attestation do
   alias LambdaEthereumConsensus.P2P
   alias LambdaEthereumConsensus.P2P.Gossip.Handler
   alias LambdaEthereumConsensus.StateTransition.Misc
+  alias LambdaEthereumConsensus.Store.Db
```

Comment:
Let's document the module and explain that this module handles the attestations from specific subnets, used by validators fulfilling aggregation duties. This is useful to not confuse it with the operations collector that handles the global attestations topic.